### PR TITLE
Patch missing MsMsInfo after merging

### PR DESCRIPTION
**Pull request by robinschmid**

### CODE REVIEW

File: src/main/java/io/github/mzmine/util/scans/SpectraMerging.java [hunk 1]
```diff
@@ -490,10 +490,26 @@ public static <T extends MassSpectrum> MergedMassSpectrum mergeSpectra(
     final int msLevel = source.stream().filter(s -> s instanceof Scan)
         .mapToInt(s -> ((Scan) s).getMSLevel()).min().orElse(1);
 
+    if (msLevel > 1) {
+      // TODO find way to merge MsMsInfo or just use a list of them?
+      var infos = source.stream().map(SpectraMerging::getMsMsInfo).filter(Objects::nonNull)
+          .findFirst().orElse(null);
+      final MsMsInfo copy = infos.createCopy();
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/util/scans/SpectraMerging.java:497]: orElse(null).createCopy

Comment (SteffenHeu) [reply src/main/java/io/github/mzmine/util/scans/SpectraMerging.java:497]: use lowest msLevel of all scans here (when merging 2, 3, 4, use 2)

**Review by SteffenHeu — CHANGES_REQUESTED** [review]

**Review by robinschmid — COMMENTED** [review]
All done

**Review by SteffenHeu — DISMISSED** [review]

**Comment by robinschmid** [comment]
Change to method reference

**Pull request merged**